ui/app/index.html

```diff
@@ -47,7 +47,7 @@
                <a ui-sref="home.streams.tabs.definitions">Streams</a>
             </li>
             <li ng-class="{ active: $state.includes('home.jobs') }">
```

Comment:
@ghillert I see this in several places now. May be it is good to have `task` in all the places (including the package/file names) given `task` is a superset now. What do you think?